src/main/java/core/basesyntax/strategy/operations/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy.operations;
+
+public interface OperationHandler {
+    int getOperationalQuantity(int quantity);
```

Comment:
```suggestion
    void handle(Transaction transaction);
```
 depending on the handler you should update your storage accordingly